.pre-commit-config.yaml

```diff
@@ -0,0 +1,49 @@
+# This file contains 2 stages
+# pre-commit stage - runs by default pre-commit
+# manual stage - runs the same tools but modifies files
+# run as: pre-commit run --all-files --hook-stage manual
+repos:
+  - repo: https://gitlab.cee.redhat.com/infosec-public/developer-workbench/tools.git
+    rev: rh-pre-commit-2.3.0
+    hooks:
+      # If you have not run this hook on your system before, it may prompt you to
+      # log in for patterns, and you will need to try again.
+      #
+      # Docs: https://source.redhat.com/departments/it/it-information-security/leaktk/leaktk_components/rh_pre_commit
+      - id: rh-pre-commit
+        name: Red Hat pre-commit (check)
+      # - id: rh-pre-commit.commit-msg # Optional for commit-msg attestation
```

Comment:
Is `gitlab.cee.redhat.com` accessible outside the firewall without a VPN connection?  If not, perhaps this section should be commented out, given that we're publishing this file on GitHub?

(I realize, that means that it cannot be used as a checkout-and-go solution, but it seems unfriendly to the greater open source community to refer to things that they cannot access. 🤷  I could be swayed either way, I guess.)